examples/with-hasura-thirdpartyemailpassword/src/Home/CallAPIView.js

```diff
@@ -3,14 +3,21 @@ import Session from "supertokens-auth-react/recipe/session";
 
 export default function CallAPIView() {
     async function callAPIClicked() {
-        let hasuraJWT = (await Session.getAccessTokenPayloadSecurely())["jwt"];
-
+        let hasuraTokenPayload = await Session.getAccessTokenPayloadSecurely();
+        let jwt = null;
+        if (hasuraTokenPayload.jwt === undefined) {
+            jwt = await Session.getAccessToken();
+        } else {
+            // This branch is only required if there are valid access tokens created before the update
+            // It can be removed after the validity period ends
+            jwt = hasuraTokenPayload.jwt;
+        }
```

Comment:
We don't really need to migration code in our examples.

```suggestion
        const jwt = Session.getAccessToken();
```